scripts/mainnet-benchmark.sh

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2034
+
+app_usage() {
+        cat <<EOF
+
+Run this like:
+
+   scripts/mainnet-benchmark.sh --epochs 3 --nix --profile time
+
+This works in two phases, prefetch and benchmark:
+
+   Phase 1:  preload the prefetcher's ChainDB, up to desired limit.
+
+   Phase 2:  run the second node as the benchmark, up to desired slot limit,
+             while feeding it blocks from the preloaded local prefetcher,
+             while also preventing said prefetcher from syncing mainnet further.
+
+App options (going BEFORE common options OR IGNORED):
+
+  Benchmark setup options:
+
+    --skip-prefetch             skip the prefetch phase
+    --prebench-pause            after prefetch, pause until 'Enter' is hit
+    --server-wait SECONDS       delay the benchmarked (client) side to start,
+                                  for this long -- to avoid profile disturbance.
+                                  Defaults to ${server_wait} seconds.
+    --no-server-wait            same as --server-wait 0
+    --node-stdout FILE          redirect stdout of benchmarked node to a file
+    --node-no-stdout            redirect stdout of benchmarked node to Dave Null
```

Comment:
That is cute and all, but I really think it should say `/dev/null` to reduce any chance of a misunderstanding.

Reply:
Agree with @erikd, we should change this to `/dev/null`.

Reply:
I was pretty sure this will attract attention : -)

Pushed.